src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -383,13 +385,13 @@ public static String getBasicInitScript(AzureVMAgentTemplate template) {
                         AzureVMManagementServiceDelegate.PRE_INSTALLED_TOOLS_SCRIPT
                                 .get(template.getBuiltInImage()).get(Constants.INSTALL_GIT));
             }
-            if (template.getBuiltInImage().equals(Constants.UBUNTU_1604_LTS) && template.getIsInstallDocker()) {
+            if (template.getIsInstallDocker()) {
```

Comment:
This block is to add a script to init scripts. Seems this image has already installed docker, so we don't need to change here.